client/src/index.scss

```diff
@@ -0,0 +1,16 @@
+body,
+html,
+#master {
+  height: 99%;
+  width: 99%;
+}
+
+body {
+  font-family: 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell',
+    'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
why need to styled body like this ?
it's mean that all pages will be centered, and that not needed always 
please move it to main page box